.github/workflows/nextjs-bundle-analysis.yml

```diff
@@ -62,18 +62,18 @@ jobs:
 
       - name: Get comment body
         id: get-comment-body
-        if: success()
+        if: success() && github.event.number
```

Comment:
Putting this back as the last PR was merged with these removed but only for testing purposes.